### ISSUE TITLE
Round Font Size to the Nearest Pt

### ISSUE THREAD

**Issue by TheJoWo** [swiadek/pixel-perfecter-sketch-plugin#10]
I'd love functionality to round the font size of text layers to the nearest pt.  Scaling can create fractional pt font sizes.

**Comment by swiadek** [comment]
Yeah, totally! Wanna give it a try? Happy to include new features via pull requests!
